src/main/java/de/tum/in/www1/artemis/domain/Authority.java

```diff
@@ -61,15 +61,15 @@ public void setName(String name) {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) {
+    public boolean equals(Object obj) {
```

Comment:
Same in this file. I'm not sure how they belong to the PR :)

Reply:
It's still an improvement, that does not need its own PR. You can simply ignore it during the review!!!